.ci/build.sh

```diff
@@ -6,16 +6,36 @@ mkdir private _site public
 
 EXPORTED_DATA="static/tasks.yaml static/instances.yaml"
 
+META_REVIEW_DATA="static/meta_review.json"
+
 if [[ -n "$GCI_TOKEN" ]]; then
   python manage.py fetch_gci_task_data private
   python manage.py cleanse_gci_task_data private _site
   rm -rf private/
 else
-  python manage.py fetch_deployed_data _site $EXPORTED_DATA
+  python manage.py fetch_deployed_data _site "$EXPORTED_DATA"
 fi
 
 python manage.py migrate
 python manage.py import_contributors_data
 python manage.py import_openhub_data
+
+# fetch deployed meta_review data
+python manage.py fetch_deployed_data _site "$META_REVIEW_DATA" True
+
+if [[ -f "$META_REVIEW_DATA" ]]; then
+  echo "File $META_REVIEW_DATA exists."
+  # Load meta_review data from json
+  python manage.py loaddata "$META_REVIEW_DATA"
+else
+   echo "File $META_REVIEW_DATA does not exist."
+fi
+
+# Run meta review system
+python manage.py run_meta_review_system
```

Comment:
split into two commands

1. for fetching external data, like `issues.json`, which goes above `python manage.py migrate`
2. processing the external data, which goes after `python manage.py migrate`, and then the dump data below it

Reply:
I am considering making use of `fetch_deployed_data` to retrieve external data (issues.json), and save it in static directory. The extra benefit is that in case `gh-board` repo breaks down, we still have raw data when debugging.

Reply:
Agree. However, saving `issues.json` on the deployed website has other benefits:
1. Other apps could reuse that file. There is no need to fetch that file again.
2. The up-to-date `issues.json` on gh-board repo may be different from what the scoring system used during deployment. 'Caching' `issues.json` makes debugging easier.

Reply:
Copies cause problems.  Think GDPR.